### ISSUE TITLE
Code style: fix modifiers ordering

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#300]
Tells that `static` should be always on start of a method then the visiblity, or the reverse. For instance:

```php
class {
    // Warning: static should be moved before public.
    public static function () { ... }
}
```

I think that we should have an option list (read-only) to specify what is the prefered order between: abstract, static and visibility (public, private and protected).

**Comment by Fleshgrinder** [comment]
The default PhpStorm is shipped with is `public static function` (or just type `pubsf` and hit tab). Hence, this should be default here, anything else would severely harm productivity, and require people to create new shortcuts.

**Comment by rentalhost** [comment]
It is not a problem... We can consider that as default ordering.

**Comment by rentalhost** [comment]
Closing: fixed on #321
